recipe/meta.yaml

```
@@ -1,40 +1,57 @@
-{% set name = "Metaphone" %}
+{% set name = "metaphone" %}
 {% set version = "0.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name[0]|upper }}{{ name[1:] }}-{{ version }}.tar.gz
   sha256: ad0beadca66cb7ec6ede71ef72bb02da097c493ddf159930d6340bc83f53da27
+  patches:
+    - patches/0001-assertEqual-over-deprecated-assertEquals.patch
 
 build:
-    number: 1005
-    script: {{ PYTHON }} -m pip install . -vv
-    noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-    host:
-        - python >=3.7
-        - pip
-        - setuptools
-    run:
-        - python >=3.7
+  build:
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
 
 test:
-    imports:
-        - metaphone
+  imports:
+    - metaphone
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - metaphone/tests
+  commands:
+    - pip check
+    # ignore these tests, they fail probably because of unicode / utf8
+    - pytest metaphone/tests --ignore metaphone/tests/test_word.py
```

```suggestion
  commands:
    - pip check
    # ignore these tests, they fail probably because of unicode / utf8
    - pytest --pyargs metaphone --ignore metaphone/tests/test_word.py
```
Because the tests are within the `metaphone` package (python package, ie `__init__.py`) and because they use `setuptools.find_packages()`, the tests will be bundled with the wheel.

So we can take advantage of `--pyargs` and make the conda package smaller and avoid duplicating the tests.

---

thanks, I missed this once again